scripts/publish.sh

```diff
@@ -5,12 +5,16 @@ cd ${OKTA_HOME}/${REPO}/packages/@okta/vuepress-site
 DEPLOY_ENVIRONMENT=""
 export REGISTRY_REPO="npm-topic"
 export REGISTRY="${ARTIFACTORY_URL}/api/npm/${REGISTRY_REPO}"
-export TEST_ENV="true"
+export PROD_ENV=""
 
 declare -A branch_environment_map
 branch_environment_map[master]=vuepress-site-prod
 branch_environment_map[staging]=vuepress-site-preprod
 
+if [[ $BRANCH == "master" ]]; then
```

Comment:
Leave a comment here indicating why we use the `PROD_ENV` variable

Reply:
Instead of PROD_ENV, wouldn't it be better to have a ENV variable with values of "test" and "prod".  This will lend better support for additional environments or if an environment name changes in the future.

Reply:
I suppose we could call this `DEPLOY_ENV` and set it to `"production"` to leave flexibility for the future